core/java8/proxy/src/main/java/org/apache/openwhisk/java/action/Proxy.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package openwhisk.java.action;
+package org.apache.openwhisk.java.action;
```

Comment:
@houshengbo @csantanapr @dgrove-oss should we add a `.runtime` subpackage here before `.java`?

Reply:
I would probably do that.  Since we're changing the package name anyways, may as well put `runtime` into it as well.

Reply:
no, we can add it behind, like org.apache.openwhisk.runtime.***